unittests/model_tests/neuron/test_synaptic_manager.py

```diff
@@ -220,14 +150,21 @@ def test_write_synaptic_matrix_and_master_population_table(self):
 
         machine_time_step = 1000.0
 
-        pre_app_vertex = SimpleApplicationVertex(10)
-        pre_vertex = SimpleMachineVertex(resources=None)
+        placements = Placements()
+        pre_app_vertex = SimpleApplicationVertex(10, label="pre")
+        pre_vertex = SimpleMachineVertex(resources=None, label="pre_m")
+        placements.add_placement(Placement(pre_vertex, 0, 0, 1))
         pre_vertex_slice = Slice(0, 9)
-        post_app_vertex = SimpleApplicationVertex(10)
-        post_vertex = SimpleMachineVertex(resources=None)
+        post_app_vertex = SimpleApplicationVertex(10, label="post")
+        post_vertex = SimpleMachineVertex(resources=None, label="post_m")
+        post_vertex_placement = Placement(post_vertex, 0, 0, 2)
+        placements.add_placement(post_vertex_placement)
         post_vertex_slice = Slice(0, 9)
-        post_slice_index = 0
-
+        delay_app_vertex = DelayExtensionVertex(
+            10, 16, pre_app_vertex, 1000, 1, label="delay")
+        delay_vertex = delay_app_vertex.create_machine_vertex(
+            post_vertex_slice, resources_required=None)
+        placements.add_placement(Placement(delay_vertex, 0, 0, 3))
         one_to_one_connector_1 = OneToOneConnector(None)
```

Comment:
This looks weird; it might be something for a different PR, but what on earth is the random_number_class argument in the OneToOneConnector(...) actually intended for?

Reply:
It seems this was for PyNN8 vs. PyNN7 again... Yes, I think this is for another PR!